src/main/java/hudson/plugins/doxygen/DoxygenArchiver.java

```diff
@@ -358,14 +353,13 @@ protected File dir() {
 		}
 	}
 
-	public MatrixAggregator createAggregator(MatrixBuild build,
-			Launcher launcher, BuildListener listener) {
-		
+	@Override
+	public MatrixAggregator createAggregator(MatrixBuild build, Launcher launcher, BuildListener listener) {
 		return new MatrixAggregator(build,launcher,listener) {
-			
-			 public boolean endBuild() throws InterruptedException, IOException {
-				 return DoxygenArchiver.this._perform(build, launcher, listener);
-			 }
+
+			public boolean endBuild() throws InterruptedException, IOException {
+				return DoxygenArchiver.this._perform(build, launcher, listener);
+			}
```

Comment:
AFAICT this is nothing but whitespace changes. Revert:

    git checkout master -- src/main/java/hudson/plugins/doxygen/DoxygenArchiver.java